snapshots/headless.js

```diff
@@ -50,20 +21,24 @@ class Renderer {
   }
 
   async start() {
-    const scriptContents = await bundleScratchblocks()
-
-    const html = `<!doctype html>
-    <meta charset=utf-8>
-    <script>${scriptContents}</script>
-    `
-
+    const app = express()
+    app.use(express.static("."))
+    await new Promise(resolve => {
+      this.server = app.listen(8000, resolve)
+    })
     this.browser = await puppeteer.launch({
       //headless: false,
       //slowMo: 250,
     })
     this.page = await this.browser.newPage()
+    this.page.on("pageerror", e => {
+      throw new Error("Page threw uncaught exception", { cause: e })
+    })
 
-    await this.page.setContent(html)
+    await this.page.goto(
+      "http://localhost:8000/snapshots/snapshot-testing.html"
+    )
+    await this.page.waitForFunction("window.scratchblocksLoaded")
```

Comment:
nit: should this be
```suggestion
    await this.page.waitForFunction(() => window.scratchblocksLoaded)
```

Reply:
Well, the docs say you can pass a string to be evaluated - so I think the current one is fine.